### DISCUSSION TITLE
WPF / .NET 5 release date?

### DISCUSSION THREAD

**Discussion by valentinba** [PrismLibrary/Prism#2455] [WPF] [answered]
Hello, is there a tentative date for when you'll publicly release the version that includes support for WPF .NET 5? I've been following the progress here Update WPF to .NET 5 #2250.
Thank you.

**Comment by brianlagunas** [comment, marked as answer]
It already supports .NET 5 because .NET 5 is .Net Core. However, we do plan on releasing 8.1 within the next week or so.
